nevergrad/functions/ml/test_mlfunctionlib.py

```diff
@@ -12,12 +12,15 @@
 
 
 def test_ml_tuning() -> None:
-    func = mlfunctionlib.MLTuning("1d_decision_tree_regression")
+    func = mlfunctionlib.MLTuning("decision_tree_depth_regression", 1)
     x: int = 3
     y1 = func(x)  # returns a float
     y2 = func(x)  # returns the same float
     np.testing.assert_array_almost_equal(y1, y2)
     y3 = func.evaluation_function(x)   # returns a float
```

Comment:
testing agains actual value would help check we do not modify the function without noticing. One test for each case with pytest framework would be perfect.
Eg:
https://github.com/facebookresearch/nevergrad/blob/099673586a544fea891c88209fd517875b56dbe2/nevergrad/functions/photonics/test_core.py#L117-L123

Reply:
Done, thx.
This also helped the code itself: the training had to be made deterministic by setting e.g. random_state=0.
Not sure how much this is a desirable feature btw.

Reply:
probably not desirable at all if you want to have variability